Updates to the ReadMe and License for this fork.

mrbiggred opened the pull request:
I've made a couple minor updates to the ReadMe but need some feedback on what other updates to make (i.e. should be remove the copyright?)  I'm also not sure what to do with the license.

mscottford commented:
The license and copyright need to stay mostly as is. We can add an additional copyright line for your modifications. Something like `Ruby 1.9 support Copyright © 2019 Chris Cumming`.

mrbiggred commented:
@mscottford I removed the copyright notice from the ReadMe file and updated the license to list Corgibytes based on this non-lawyer advice I found on the Internet.  Let me know if you want anything else changed.  Thanks.

https://opensource.stackexchange.com/questions/4439/how-to-deal-with-licences-after-forking-a-project
